cmake/templates/conf.py.in

```diff
@@ -517,7 +517,7 @@ master_doc = 'index'
 
 # General information about the project.
 project = u'HPX'
-copyright = u'2008-2020, The STE||AR Group'
+Copyright = u'2008-2020, The STE||AR Group'
```

Comment:
`copyright` is a sphinx configuration variable, so `Copyright` won't do. However, it seems like `project_copyright` has been added as an alternative (probably exactly for the reason that you've opened this PR) so I think you can rename the variable to that instead. See https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-project_copyright.

Reply:
Hmm, didn't know that. Changed to `project_copyright`.